frontend/storybook/src/components/entry/homepage/plain-text-button.ts

```diff
@@ -0,0 +1,35 @@
+import "@elements/core/buttons/text";
```

Comment:
this file seems rather pointless? button-text is tested elsewhere

Reply:
Sure, can be deleted

Reply:
Actually, ideally to be kept so it can be added in as a function in other components. Already being used in one place.